server/testsupport/ciagInductionTestDataBuilder.ts

```diff
@@ -0,0 +1,60 @@
+import type { CiagInduction } from 'ciagInductionApiClient'
+
+const aCiagInductionWithNoRecordOfAnyPreviousWorkExperience = (prisonNumber = 'A1234BC'): CiagInduction => {
+  return {
+    ...baseCiagInductionUsedForBuildingOtherInstances(prisonNumber),
+    workExperience: null,
+  }
+}
+
+const aCiagInductionWithNoPreviousWorkExperience = (prisonNumber = 'A1234BC'): CiagInduction => {
+  return {
+    ...baseCiagInductionUsedForBuildingOtherInstances(prisonNumber),
+    workExperience: {
+      hasWorkedBefore: false,
+      modifiedBy: 'ANOTHER_DPS_USER_GEN',
+      modifiedDateTime: '2023-08-22T11:12:31.943Z',
+      workExperience: null,
+    },
+  }
+}
+
+const aCiagInductionWithPreviousWorkExperience = (prisonNumber = 'A1234BC'): CiagInduction => {
+  return {
+    ...baseCiagInductionUsedForBuildingOtherInstances(prisonNumber),
+    workExperience: {
+      hasWorkedBefore: true,
+      modifiedBy: 'ANOTHER_DPS_USER_GEN',
+      modifiedDateTime: '2023-08-22T11:12:31.943Z',
+      workExperience: [
+        {
+          typeOfWorkExperience: 'CONSTRUCTION',
+          role: 'General labourer',
+          details: 'Groundwork and basic block work and bricklaying',
+        },
+        {
+          typeOfWorkExperience: 'OTHER',
+          otherWork: 'Retail delivery',
+          role: 'Milkman',
+          details: 'Self employed franchise operator delivering milk and associated diary products.',
+        },
+      ],
+    },
+  }
+}
+
+const baseCiagInductionUsedForBuildingOtherInstances = (prisonNumber = 'A1234BC'): CiagInduction => {
```

Comment:
Not sure about the name - perhaps it could have `template` in it somewhere?

I think it would be useful to have a method called something like `aCiagInductionWithMinimumData()`, but we don't know what that looks like yet!

Anyway, ignore for this now. I know you're keen to get it merged, so I'll look into it on my branch.

Reply:
Yes, we can defo rename / rework this stuff moving forwards 👍